atd-vze/src/views/Crashes/crashDataMap.js

```diff
@@ -94,23 +93,25 @@ export const createCrashDataMap = isTempRecord => {
           label: "Suspected Serious Injury Count",
           relationshipName: "crash_injury_metrics_view",
           editable: false,
-          uiType: "text",
         },
         vz_fatality_count: {
           label: "ATD Fatality Count",
           relationshipName: "crash_injury_metrics_view",
           editable: false,
-          uiType: "text",
         },
         cris_fatality_count: {
-          label: "CRIS Death Count",
+          label: "CRIS Fatality Count",
           relationshipName: "crash_injury_metrics_view",
           editable: false,
         },
-        law_enforcement_fatality_num: {
+        law_enf_fatality_count: {
           label: "Law Enforcement Fatality Count",
+          relationshipName: "crash_injury_metrics_view",
+          editable: false,
+        },
+        law_enforcement_fatality_num: {
+          label: "Law Enforcement YTD Fatal Crash",
           editable: true,
-          uiType: "text",
```

Comment:
the fields look good, but when i try to edit this field the input is blank. i think the `uiType` property needs to come back

<img width="614" alt="Screenshot 2024-06-03 at 1 53 48 PM" src="https://github.com/cityofaustin/atd-vz-data/assets/14793120/2dc11a94-57fc-44a5-8360-61bcd05d9f67">

Reply:
fixed this, also curious why the `law_enforcement_fatality_num` is of type `text` in the database instead of `int`?

Reply:
yes—thanks for checking on that. there are two text values in the the old `law_enforcement_num` in prod. i reached out to Xavier and Mary about those because i would also like to switch it to integer. it's on the todos tracker so i think we can just roll with this for now 👍